matrix/test_unittest_matrix.py

```diff
@@ -157,16 +149,12 @@ def test_not_equal(self):
 
 
 class TestMatrixIteration(unittest.TestCase):
```

Comment:
why you removed the classes docs?

Reply:
Thought you meant to remove them in a previous comment. Should I leave the docs of the methods as they are? The names are pretty self-explanatory.

Reply:
In general any class and function should have some documentation (the only exception is built in functions)